api/v1/constructors.go

```diff
@@ -1074,15 +1076,6 @@ func NewSystemSpec(systemInfo v1info.SystemInfo) (*SystemSpec, error) {
 	}
 
 	spec.VSwitchType = &systemInfo.Capabilities.VSwitchType
-
-	if systemInfo.DRBD != nil {
```

Comment:
Same comment as the above two blockes

Reply:
I have undo the changes from constructor and updated the code of build filters